Directory.Build.targets

```diff
@@ -16,4 +16,6 @@
       <_EmbedFileResfilePath>$(IntermediateOutputPath)$(AppxSubfolderWithFilesToBeEmbedded)\embed.resfiles</_EmbedFileResfilePath>
     </PropertyGroup>
   </Target>
+
+  <Target Name="ValidateWinUIPlatform" />
```

Comment:
This is to override the checks in the targets. Things work a bit now since the dll is back to an Any CPU platform. However, there seems to be logic in the targets files to copy files out of the `runtimes/win-$(Platform)/native` folder, which should happen automatically.